lib/backend/api/index.js

```diff
@@ -9,7 +9,7 @@ const _ = require('lodash')
 
 module.exports = function (config) {
   const { common } = config
-  const { db, env } = common.storage
+  const { db, env, subsub } = common.storage
```

Comment:
Typo? should be `pubsub`

Reply:
Yes, this is true indeed, I wonder how this works then.. let me check logs :D oh, I see, that part I didn't copy.